src/sage/geometry/hyperplane_arrangement/arrangement.py

```diff
@@ -3560,13 +3615,13 @@ def derivation_module_basis(self, algorithm="singular"):
             ....:     else:
             ....:         assert exponents(B) == exponents(Bp)
         """
-        alg = algorithm # prevent possible changes to a global variable
+        alg = algorithm  # prevent possible changes to a global variable
         if alg == "singular":
             # import sage.libs.singular.function_factory
             # syz = sage.libs.singular.function_factory.ff.syz
             f = self.defining_polynomial()
-            I = f + f.jacobian_ideal()
-            IS = I._singular_()
+            I0 = f + f.jacobian_ideal()
+            IS = I0._singular_()
```

Comment:
For future reference, this kind of change (which isn't actually necessary as there is no danger of confusion between `l` and `I` here; it's not a good idea to blindly apply style "rules") really should be done on a separate PR in general. This increases the chances of merge conflicts and it makes it harder to track down if bugs are introduced (the PR subject isn't related to the change). Of course, in more localized ways, this is fine, but not en masse.

Reply:
My bad; do you think it must be undone?